### PR TITLE
Managing Throws

### PR DESCRIPTION
**Pull request by powercycle**
This is a first whack at talking about how throws can be managed.  Tried to use a very simple example.

I feel this is sort of weak in the description area so please make suggestions for improvement.

### CODE REVIEW

**Comment by powercycle** [comment]
Do I even need to make request for reviewers, or is this somehow already automatic?

File: advanced/managing-throws.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+For example, if you only want the aileron to travel a maximum of 50% and have the stick be able to travel the entire distance of its physical movement, just change the Min and Max numbers to 50.
+
+![](/assets/Outputs-50.png)
+
+There is no way to assign a switch to to this setting, however, and the limits will apply to all flight modes.
+
+A good example for when you would use this method is that even though an elevator could physically travel a full inch, doing that would make the plane behave badly and out of control.  Setting output limits would reduce the possibility of the elavator traveling too far no matter what you put into the Inputs or Mixer screens.
+
+
```
Comment (schwabe) [inline advanced/managing-throws.md]: Maybe add a sentence here: It is always good pratice to set the physical limits in the output as it reduces the chance of accidently damaging the model while setting up mixer/inputs?

Comment (powercycle) [reply advanced/managing-throws.md]: OK.  will do.  Think I'll just change the last sentence to incorporate your idea.

**Comment by schwabe** [comment]
Thanks!

**Pull request merged**